src/vs/workbench/browser/parts/editor/singleEditorTabsControl.ts

```diff
@@ -67,12 +68,12 @@ export class SingleEditorTabsControl extends EditorTabsControl {
 		this._register(toDisposable(() => titleContainer.classList.remove('breadcrumbs'))); // important to remove because the container is a shared dom node
 
 		// Right Actions Container
-		const actionsContainer = document.createElement('div');
-		actionsContainer.classList.add('title-actions');
-		titleContainer.appendChild(actionsContainer);
+		this.actionsContainer = document.createElement('div');
```

Comment:
This change is not needed anymore I think and can go back to `const actionsContainer...`